test/test_entry.py

```diff
@@ -31,6 +31,60 @@ def testEntry(self):
         pass
 
 
+class TestDefinitionOfSenses(unittest.TestCase):
+    def setUp(self) -> None:
+        pass
+
+    def tearDown(self) -> None:
+        pass
+
+    def test_definition_of_sense_init(self) -> None:
+        evaluator = DefinitionOfSenseEvaluator()
+
+        self.assertEqual(evaluator.senses_count, 0)
+        self.assertEqual(evaluator.definition_count, 0)
+        self.assertEqual(evaluator.entry_count, 0)
+
+    def test_reset(self) -> None:
+        evaluator = DefinitionOfSenseEvaluator()
+        f = open("data/entries.json")
```

Comment:
For the test to pass, this path needs to be `test/data/entries.json`

Reply:
adapted the path